pkg/tenant/members.go

```diff
@@ -83,21 +125,30 @@ func (s *Server) TenantMemberCreate(c *gin.Context) {
 	c.JSON(http.StatusCreated, out)
 }
 
+// MemberList retrieves all members assigned to an organization
+// and returns a 200 OK response.
+//
+// Route: /member
 func (s *Server) MemberList(c *gin.Context) {
-	// The following TODO task items will need to be
-	// implemented for each endpoint.
+	// TODO: Fetch the member's tenant ID from key.
+	var tenantID ulid.ULID
+
+	// Get members from the database and return a 500 response
+	// if not succesful.
+	if _, err := db.ListMembers(c.Request.Context(), tenantID); err != nil {
+		log.Error().Err(err).Msg("could not fetch members from database")
+		c.JSON(http.StatusInternalServerError, api.ErrorResponse("could not fetch members from database"))
+		return
+	}
+
+	// Build the response.
+	out := &api.MemberPage{Members: make([]*api.Member, 0)}
 
-	// TODO: Add authentication and authorization middleware
-	// TODO: Identify top-level info
-	// TODO: Parse and validate user input
-	// TODO: Perform work on the request, e.g. database interactions,
-	// sending notifications, accessing other services, etc.
+	member := &api.Member{}
 
-	// Return response with the correct status code
+	out.Members = append(out.Members, member)
```

Comment:
So do we need to do the same thing here that we did in `TenantMemberList`? Loop through each member model and construct the `api.Member`?

Reply:
A follow on story was created to filter the list by the `Tenant ID` once the authentication middleware was available, so I decided to make the change you've mentioned here when I work on that. Now that the middleware has been created, I'll work on that soon.